package.json

```diff
@@ -63,21 +64,27 @@
     "webpack-dev-server": "^2.9.7"
   },
   "jest": {
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/config/webpack/test.js"
-    ],
-    "setupFiles": [
-      "./app/javascript/support/requestAnimationFrameTestShim.js"
-    ],
+    "testPathIgnorePatterns": ["/node_modules/", "/config/webpack/test.js"],
+    "setupFiles": ["./app/javascript/support/requestAnimationFrameTestShim.js"],
     "setupTestFrameworkScriptFile": "./app/javascript/support/testSetup.js"
   },
   "scripts": {
-    "lint": "npm run lint:styles && npm run lint:js",
-    "lint:js": "eslint ./app ./config ./spec *.js",
-    "lint:styles": "sass-lint -c .sass-lint.yml 'app/assets/stylesheets/**/*.scss' -v -q",
+    "lint": "yarn lint:styles && yarn lint:js && yarn lint:prettier",
+    "lint:js": "eslint ./app ./spec *.js",
+    "lint:prettier":
+      "yarn prettier:debug:scss && yarn prettier:debug:js && yarn prettier:debug:json",
+    "lint:styles":
+      "sass-lint -c .sass-lint.yml 'app/assets/stylesheets/**/*.scss' -v -q",
+    "prettier":
+      "prettier README.md package.json .eslintrc ./app/**/*.js ./app/**/*.scss --write",
```

Comment:
Formatting `package.json` could conflict with how npm/yarn formats it. It might be best not to run prettier on it.